packages/mds-transactions-api/handlers/get-transactions.ts

```diff
@@ -0,0 +1,26 @@
+import { TransactionServiceClient, TransactionDomainModel } from '@mds-core/mds-transactions-service'
+import { ApiRequestParams } from '@mds-core/mds-api-server'
+import { parseRequest } from '@mds-core/mds-api-helpers'
+import { ServerError } from '@mds-core/mds-utils'
+import { TransactionApiRequest, TransactionApiResponse } from '../@types'
+
+export type TransactionApiGetTransactionsRequest = TransactionApiRequest &
+  ApiRequestParams<'provider_id' | 'start_timestamp' | 'end_timestamp'>
+
+export type TransactionApiGetTransactionsResponse = TransactionApiResponse<{ transactions: TransactionDomainModel[] }>
+
+export const GetTransactionsHandler = async (
+  req: TransactionApiGetTransactionsRequest,
+  res: TransactionApiGetTransactionsResponse
+) => {
+  try {
+    const transactions = await TransactionServiceClient.getTransactions({
+      ...parseRequest(req).single({ parser: String }).query('provider_id'),
+      ...parseRequest(req).single({ parser: Number }).query('start_timestamp', 'end_timestamp')
```

Comment:
Should have a `last_id` or something similar for cursor paging, or limit the timestamp range to 30 days or similar if we want to add pagination later.

Reply:
yeah, pagination is unaddressed at the moment.  let's discuss once you get your app talking to the api.